src/Symfony/Component/VarDumper/Tests/Caster/CasterTest.php

```diff
@@ -185,4 +185,31 @@ public function __debugInfo(): array
             }
         });
     }
+
+    public function testClassHierarchy()
+    {
+        $c = eval(<<<'PHP'
```

Comment:
I don't think we need `eval` for that test. Create actual fixtures instead.

Reply:
Fixed.